### PR TITLE
feat(timeseriescard): expose the thresholds for x and y axis for time series chart

### PR DESCRIPTION
**Pull request by scottdickerson**
Closes #

**Summary**

- Carbon charts has added a thresholds prop and we're exposing this now in the TimeSeriesCard

**Change List (commits, features, bugs, etc)**

- fix(package.json): add helper builder react jsx to support builds, they started breaking
- fix(storybooksnapshots): mock the document and window for getComputedStyle
- feat(TimeSeriesCard): add thresholds prop

**Acceptance Test (how to verify the PR)**

- view the thresholds story of the TimeSeriesCard

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify a few of the other TimeSeriesCard stories

### CODE REVIEW

File: packages/react/src/components/StorybookSnapshots.test.js [hunk 1]
```diff
@@ -87,6 +87,10 @@ describe(`Storybook Snapshot tests and console checks`, () => {
     // https://stackoverflow.com/questions/53271193/typeerror-scrollintoview-is-not-a-function
     window.HTMLElement.prototype.scrollIntoView = jest.fn();
     window.HTMLElement.prototype.scrollTo = jest.fn();
+    window.getComputedStyle = () => ({
+      getPropertyValue: () => '25',
+    });
+    window.document = new Document();
```
Comment (kevinsperrine) [inline packages/react/src/components/StorybookSnapshots.test.js:93]: We should reset these after the tests.

File: packages/react/package.json [hunk 2]
```diff
@@ -166,6 +166,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.16",
+    "@babel/helper-builder-react-jsx": "^7.12.13",
```
Comment (kevinsperrine) [inline packages/react/package.json:169]: Any idea why this was needed? It doesn't come up in a yarn why. Just curious what's causing the need for it now.

**Review by kevinsperrine — APPROVED** [review]
Just a couple notes.

File: packages/react/package.json (hunk 2 above)
Comment (scottdickerson) [reply packages/react/package.json:169]: I'm not sure, I'm getting build failures with a failure to resolve that package

**Review by bjornalm — APPROVED** [review]
LGTM

File: packages/react/src/components/StorybookSnapshots.test.js (hunk 1 above)
Comment (davidicus) [reply packages/react/src/components/StorybookSnapshots.test.js:93]: We should be good to go once this is addressed.

Comment (scottdickerson) [reply packages/react/src/components/StorybookSnapshots.test.js:93]: thanks for catching, this is resolved now @davidicus @kevinsperrine

**Review by herleraja — APPROVED** [review]
Lgtm

**Review by sstone2423 — APPROVED** [review]

**Review by davidicus — APPROVED** [review]

**Pull request merged**